src/init/features/storage.ts

```diff
@@ -1,17 +1,15 @@
-"use strict";
+import * as clc from "cli-color";
```

Comment:
How come github was able to understand I ts-ified this file but not the others? :/

Reply:
git has some kind of threshold where renaming plus changing a bunch of things no longer looks like the same file, I don't think there's anything to be done about it :man_shrugging: